float64 AMSLAltAboveTerrain
float64 Altitude
int32 AltitudeMode
bool autoContinue
int32 command
int32 doJumpId
int32 frame
float64[] params
string type

